types/validator_set.go

```diff
@@ -4,13 +4,17 @@ import (
 	"bytes"
 	"fmt"
 	"math"
+	"math/big"
 	"sort"
 	"strings"
 
 	"github.com/tendermint/tendermint/crypto/merkle"
 	cmn "github.com/tendermint/tendermint/libs/common"
 )
 
+// The maximum allowed total voting power.
+const MaxTotalVotingPower = 8198552921648689607
```

Comment:
where's this number coming from?

Reply:
Thanks for the review Anton. It's the max int64 `x` where `x + (x >> 3)` still stays in bounds of int64.

Reply:
Thanks! Could you please add this as a comment?

Reply:
Not sure, I didn't hink very long about it but the formula would be 
`math.MaxInt64 / 1.125` or `math.MaxInt64 / (1. + 1./8.)` which instead yields `8198552921648689000` which still isn't the max allowed number. Maybe there is a way using the shift operator I'm not seeing. 
Probably 8198552921648689000 instead of 8198552921648689607 is still more than enough for an upper bound though.